products/contour.md

```diff
@@ -20,39 +20,45 @@ identifiers:
 
 # eol(X) = releaseDate(X+3)
 releases:
+-   releaseCycle: "1.30"
+    releaseDate: 2024-08-01
```

Comment:
```suggestion
    releaseDate: 2024-07-31
```

According to https://github.com/projectcontour/contour/releases/tag/v1.30.0 (normalize to GMT, the timezone used by our automation).